lib/plugins/TaskRunner/Drupal.js

```diff
@@ -0,0 +1,171 @@
+"use strict"
+
+module.exports = class Drupal extends require('./Script') {
+
+
+  /**
+   * Options (used by this task):
+   *   - clearCaches: (boolean) - Whether to clear all caches after the build is finished. Defaults to true.
```

Comment:
Yeah, see the `options.clearCaches` example above. See http://usejsdoc.org/tags-param.html#parameters-with-properties for a few examples.